src/Form/BaseEventCommandType.php

```diff
@@ -58,4 +62,31 @@ public function configureOptions(OptionsResolver $resolver)
             ])
         ;
     }
+
+    /**
+     * Creates the begin date in the future for the next minute step.
+     */
+    private function createBeginDate(array $minutes): \DateTime
+    {
+        $now = new \DateTime();
+        $nowMinute = $now->format('i');
+        $step = reset($minutes);
```

Comment:
to initialise $step variable if $minutes array is empty

Reply:
In that case, `$step` should be initialized to `null` and conditions below should be modified with something like :

```php
      if ($step === null) {
           return $now;
      }

      if ($step < $nowMinute) {
            return $now
                ->modify('next hour')
                ->modify(sprintf('-%d minutes', $nowMinute))
            ;
     }

    return $now->modify(sprintf('+%d minutes', $step - $nowMinute));
```

WDYT ?

Reply:
I initialise `$step` with `null` and if `$minutes` is empty then the returned time will be the next hour because it valides `$step < $nowMinute` condition.

ex: if current time is 16h05 and `$minutes` is empty, and `$step` is null, then returned time will be 17h00.
ok for you?

Reply:
Yes yes yes :1st_place_medal:

Reply:
in this case it will be the same time, no pb :)